srcpkgs/xfce4-panel-profiles/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'xfce4-panel-profiles'
+pkgname=xfce4-panel-profiles
+version=1.0.13
+revision=1
+build_style=configure
+configure_args="--prefix=/usr --python=python3"
+hostmakedepends="intltool tar"
```

Comment:
why does this need tar?

Reply:
The install procedure requires it for the layouts as far as I can tell. Without it on the host system, it fails:

```
...
cd data/layouts/cupertino; tar  -cvjf "../Cupertino.tar.bz2" *
/bin/sh: line 1: tar: command not found
make: *** [Makefile:36: layouts] Error 127
```

Reply:
ok just wanted to check